onadata/apps/api/viewsets/xform_list_viewset.py

```diff
@@ -73,15 +73,22 @@ def get_renderers(self):
 
     def filter_queryset(self, queryset):
         username = self.kwargs.get('username')
-        if username is None and self.request.user.is_anonymous:
+        form_uuid = self.kwargs.get('form_uuid')
+        if (username is None and form_uuid is None) and \
```

Comment:
do we need `is None`? `not` will work unless we want this to be False when `username` is False

Reply:
I think using `not` here is enough. I've changed this in the latest commits.